### PR TITLE
#157731607 Format code to PEP8 & Flake8 python style standard

### PR DESCRIPTION
**Pull request by hariclerry**
### What does this PR do?
Format code to meet PEP8 python coding style.

#### Description of Task to be completed?
1. Install flake8 and fix all pep8 issues.

#### How should this be manually tested?
1. Run the commands below
```
$ git clone https://github.com/andela/wg-rogue-one.git
$ cd wg-rogue-one

$ git checkout ch-implement-python-coding-style-157731607

```
2. follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ch-implement-python-coding-style-157731607/README.rst)
3. run the command ```pip install requirements.txt file```
4. run the command ```autopep8 -i -r --global-config setup.cfg wg-rogue-one```
5. run the command ```flake8 --config setup.cfg  wg-rogue-one```

#### Any background context you want to provide?
To check for Pep8 issues, run the above commands with specific file name or folder name.
  
#### What are the relevant pivotal tracker stories?
#157731607

### CODE REVIEW

**Comment by hariclerry** [comment]
Hey guys, can you please review my code and let me know what you think. Am still fixing a few issues though.

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,8 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.DS_Store
```
Comment (tibetegya) [inline .gitignore]: put a * before .DS_Store since some instances of that file are nested in other folders

File: requirements.txt [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
-
-# REST API
-djangorestframework>=3.2,<3.3
-django-filter
-django-tastypie
-django-cors-headers
-
-# Python3 compatibility
-six
+alabaster==0.7.10
```
Comment (tibetegya) [inline requirements.txt]: I recommend you only add the packages you manually installed, their dependancies will automatically be installed, you do not have to add them here.

File: setup.cfg [hunk 3]
```diff
@@ -5,3 +5,8 @@ universal = 1
 exclude = urls.py,tasks.py,*migrations*,*bower_components*
 max-line-length = 100
 ignore = W503
+
+[flake8]
+exclude = .git,__pycache__,urls.py,tasks.py,*migrations*,*bower_components*,*docs*,__*__,
+max-line-length = 100
+ignore = F401,W503
```
Comment (tibetegya) [inline setup.cfg]: add a new line at the end of your files

Comment (hariclerry) [reply setup.cfg]: @tibetegya, there is a new line but I don't know why it is not being shown here.

File: requirements.txt (hunk 2 above)
Comment (hariclerry) [reply requirements.txt]: I will work on that.

File: .gitignore (hunk 1 above)
Comment (dondrzzy) [reply .gitignore]: Plus 1

**Review by dondrzzy — COMMENTED** [review]
Some good work Harriet. This shows that pep8 is doing its work.

**Comment by emugaya** [comment]
@hariclerry push your code to GitHub again for the build to pass

**Comment by hariclerry** [comment]
@dondrzzy, great. @emugaya, I have pushed to Github again and hope the build passes.

**Review by emugaya — CHANGES_REQUESTED** [review]
Resolve merge conflicts

**Comment by hariclerry** [comment]
@emugaya, merge conflicts resolved.

**Comment by emugaya** [comment]
@hariclerry I have failed to test your PR using the instructions you give on the PR. Which folder should be in to run these commands?

**Comment by hariclerry** [comment]
@emugaya, you first run the commands on the wger folder first e.g, 
autopep8 -i -r --global-config setup.cfg wger
flake8 --config setup.cfg wger

but the commands should work on the entire project folder.

**Comment by hariclerry** [comment]
@emugaya, I have also updated the PR.

**Comment by emugaya** [comment]
@hariclerry What is the output I should get after running the above commands? `autopep8 -i -r --global-config setup.cfg wger` does not give any output.

**Comment by hariclerry** [comment]
@emugaya, when you run the commands and do not see any output, it means there are no issues/error with the pep8 standard. It means all is well. For testing purpose, open the setup.cfg file and remove docs folder under the flake8 config settings and run the command for flake8 again, you should be able to see some error messages displayed as output in the terminal.

**Review by emugaya — APPROVED** [review]

**Pull request merged**